SpacetimeDB.ClientSDK.csproj

```diff
@@ -19,19 +19,31 @@
     <AssemblyVersion>0.12.0</AssemblyVersion>
     <Version>$(AssemblyVersion)</Version>
     <DefaultItemExcludes>$(DefaultItemExcludes);*~/**</DefaultItemExcludes>
+    <!-- Disable default compile items to prevent duplicate inclusion -->
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <!-- We want to save DLLs for Unity which doesn't support NuGet. -->
     <RestorePackagesPath>packages</RestorePackagesPath>
   </PropertyGroup>
 
+  <!-- Package references -->
   <ItemGroup>
     <PackageReference Include="SpacetimeDB.BSATN.Runtime" Version="0.12.*" />
-
     <InternalsVisibleTo Include="SpacetimeDB.Tests" />
   </ItemGroup>
 
+  <!-- Include non-code files -->
   <ItemGroup>
     <None Update="logo.png" Pack="true" PackagePath="" />
     <None Update="README.md" Pack="true" PackagePath="" />
   </ItemGroup>
 
+  <!-- Explicitly include all .cs files except Unity-related scripts -->
+  <ItemGroup>
+    <Compile Include="src/**/*.cs" Exclude="unity-tests/client/Assets/**/*.cs" />
```

Comment:
This prevents the unity testsuite cs files from being compiled during the dotnet testsuite.